mappings/net/minecraft/client/texture/NativeImage.mapping

```diff
@@ -198,6 +204,16 @@ CLASS net/minecraft/class_1011 net/minecraft/client/texture/NativeImage
 	METHOD method_4327 loadFromTextureImage (IZ)V
 		ARG 1 level
 		ARG 2 removeAlpha
+	METHOD method_47594 copyRect (Lnet/minecraft/class_1011;IIIIIIZZ)V
+		ARG 1 image
+		ARG 2 x
+		ARG 3 y
+		ARG 4 translateX
+		ARG 5 translateY
```

Comment:
These are destination coordinates, not the difference between the original position and the target position.

Reply:
Fixed